### PR TITLE
standard-js

### PR DESCRIPTION
**Pull request by aricart**
adoption of standard-js for the formatting.
this change removes the use of url.parse() (bumping to v2.0.0)
bumped min requirements for node to 10, matching currently supported versions

### CODE REVIEW

File: benchmark/pub_perf.js [hunk 1]
```diff
@@ -13,52 +13,49 @@
...
 
-const NATS = require('../lib/nats');
-const nats = NATS.connect();
-const fs = require('fs');
+const NATS = require('../lib/nats')
+const nats = NATS.connect()
+const fs = require('fs')
 
-///////////////////////////////////////
+/// ////////////////////////////////////
```
Comment (derekcollison) [inline benchmark/pub_perf.js]: This looks weird, why did it add that?

File: benchmark/pub_sub_perf.js [hunk 2]
```diff
@@ -13,56 +13,54 @@
...
-const NATS = require('../lib/nats');
-const nc1 = NATS.connect();
-const nc2 = NATS.connect();
+const fs = require('fs')
+const NATS = require('../lib/nats')
+const nc1 = NATS.connect()
+const nc2 = NATS.connect()
 
-///////////////////////////////////////
+/// ////////////////////////////////////
```
Comment (derekcollison) [inline benchmark/pub_sub_perf.js]: Same here.

File: benchmark/reconnect_perf.js [hunk 3]
```diff
@@ -13,62 +13,61 @@
...
-const fs = require('fs');
-const NATS = require('../lib/nats'),
-    nsc = require('../test/support/nats_server_control');
+const fs = require('fs')
+const NATS = require('../lib/nats')
+const nsc = require('../test/support/nats_server_control')
 
-
-///////////////////////////////////////
+/// ////////////////////////////////////
```
Comment (derekcollison) [inline benchmark/reconnect_perf.js]: Confused why it keeps doing this.

File: benchmark/pub_perf.js (hunk 1 above)
Comment (aricart) [reply benchmark/pub_perf.js]: @derekcollison 'spaced-comment' rule:
https://eslint.org/docs/rules/spaced-comment

The standardjs is opinionated like go, and they don't like that :(

Comment (derekcollison) [reply benchmark/pub_perf.js]: ok

**Comment by aricart** [comment]
@derekcollison lowered version to 1.3.6, as `new URL()` is available since 7.0.0, I am going to do a test, I might have to add an import instead of using a global object.

File: benchmark/pub_perf.js (hunk 1 above)
Comment (kozlovic) [reply benchmark/pub_perf.js]: Yes, but you could still replace with another valid comments maybe?
Like:
```
/*
 * Publish Performance
 *
 */
```
or some others in the list of "correct" comments?

Comment (kozlovic) [reply benchmark/pub_perf.js]: Or this maybe?
```
//--------------
// Publisher Performance
//---------------
```

**Comment by aricart** [comment]
Yes - changed so that we `require('url')` and do `new url.URL()` as this is compatible all the way back to node 7.

**Pull request merged**